roles/edpm_accel_drivers/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+edpm_accel_drivers_blacklist_nouveau: true
```

Comment:
@bogdando  - By default, only the blacklist/modprobe tasks will run. We can augment this as necessary to support passthrough use cases.